src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1117,6 +1117,7 @@ protected override void GetItem(string path)
 
             try
             {
+#if !UNIX
                 bool retrieveStreams = false;
                 FileSystemProviderGetItemDynamicParameters dynamicParameters = null;
```

Comment:
`Copy-Item` doesn't expose a `-Stream` param at all.  In the case of copying a NTFS file that contains alternate streams, those streams don't exist on the Linux destination.